src/components/Admin/BatchContent.svelte

```diff
@@ -0,0 +1,206 @@
+			console.error(error);
+			throw svelteError(500, 'Server error while uploading to Contentful. Try again later.');
+		}
+		isContentfulEntriesLoading = false;
+	}
+</script>
+
+<form class="mx-auto flex w-max  flex-col items-center justify-center pt-8">
+	<h1 class="mb-4 font-display text-5xl">Found a new cool AI tool?</h1>
+	<p class="mb-8">Let's add it to Mosaic!</p>
+
+	<div class="mb-8 w-max">
+		<p class="mb-4 text-center font-display">How do you want to summarize this website?</p>
+
+		<div class="grid w-full grid-cols-3 gap-2">
+			<label class="rounded border-3 border-black bg-white px-5 py-4">
+				<input
+					type="radio"
+					bind:group={generationMethod}
+					name="method"
+					value="cached-content"
+					checked />
+				<span>Cached Content</span>
+			</label>
+
+			<label class="rounded border-3 border-black bg-white px-5 py-4">
+				<input type="radio" bind:group={generationMethod} name="method" value="content" />
+				<span>Website's Content</span>
+			</label>
+
+			<label class="rounded border-3 border-black bg-white px-5 py-4">
+				<input type="radio" bind:group={generationMethod} name="method" value="url" />
+				<span>URL</span>
+			</label>
+		</div>
+	</div>
+
+	<label class="c-field w-full">
+		<span class="c-field-label">List of URLs, comma separated</span>
+		<textarea
+			class="c-field-input"
+			name="urls"
+			bind:value={urls}
+			placeholder="https://example1.com, https://example2.com"
+			required />
+	</label>
+
+	<div class="mb-12 w-full">
+		<button
+			on:click={handleSiteContentClick}
+			type="button"
+			class="c-btn-submit w-full disabled:cursor-not-allowed disabled:opacity-20"
+			disabled={isLoading}>
+			Get Site Content (Readiblity + Meta Description)
+		</button>
+	</div>
+</form>
+
+{#if !isSiteContentLoading && allSiteReadbilityAndMetaDescriptions.length > 0}
+	<div class="mx-auto flex flex-col items-center justify-center gap-8 pt-8">
+		<h1 class="mb-4 font-display text-5xl">Site Content</h1>
+		<p class="mb-8">Here is the content that was generated for each site</p>
+		{#each allSiteReadbilityAndMetaDescriptions as { readability, metaDescription }, i (readability.url)}
+			<BatchResults
+				bind:allAiContent
+				bind:allSiteReadbilityAndMetaDescriptions
+				{i}
+				{readability}
+				{metaDescription} />
+		{/each}
+	</div>
+{/if}
+
+{#if !isContentfulEntriesLoading && allContentfulEntries.length > 0}
+	<div class="mx-auto flex w-max  flex-col items-center justify-center pt-8">
+		<h1 class="mb-4 font-display text-5xl">Contentful Entries</h1>
+		<p class="mb-8">Here are the entries that were created in Contentful</p>
+		<div class="grid w-full grid-cols-3 gap-2">
+			{#each allContentfulEntries as entry}
+				<div class="max-w-xs rounded border-3 border-black bg-white px-5 py-4">
+					<h2 class="mb-4 text-lg font-bold">{entry.fields.name['en-US']}</h2>
+					<p>{entry.fields.description['en-US']}</p>
+					<p class="my-4 font-display">{entry.fields.url['en-US']}</p>
```

Comment:
Can we add `break-all` here so URLs don't overflow the card? 😬

<img width="406" alt="image" src="https://user-images.githubusercontent.com/11279377/232576097-2afc8374-3dac-4244-baed-65e7160955de.png">